module/VuFind/src/VuFind/RecordDriver/Factory.php

```diff
@@ -201,7 +201,33 @@ public static function getSolrMarcRemote(ServiceManager $sm)
         $driver->attachSearchService($sm->get('VuFindSearch\Service'));
         return $driver;
     }
+    
+    /**
+     * Factory for SolrOverdrive record driver.
+     *
+     * @param ServiceManager $sm Service manager.
+     *
+     * @return SolrOvedrive
+     */
+    public static function getSolrOverdrive(ServiceManager $sm)
```

Comment:
I recently refactored all of the record driver factories to use the new style; this will have to get moved. Sorry for the nuisance!

Reply:
Should the new factory necessarily extend the SolrMarcFactory if the RecordDriver extends SolrMarc?  the constructor arguments are different since it uses my connector instead of the TitleHolds logic stuff.

Reply:
No, the factory doesn't have to extend SolrMarcFactory if that doesn't make sense under the circumstances. If it's easier/more logical to make a separate factory, that's fine.